src/renderer/components/dialogs/Settings.tsx

```diff
@@ -57,6 +57,50 @@ export function SettingsSelector(props: any) {
   )
 }
 
+function DeltaInput({
```

Comment:
Hmm we already have a component called DeltaInput in LoginStyles. Can you either fuse them or give it nother name?